pkgs/development/python-modules/tabula-py/default.nix

```diff
@@ -39,11 +36,15 @@ buildPythonPackage rec {
     setuptools-scm
```

Comment:
Add setuptools, as you're currently relying on setuptools-scm propagating it.

https://github.com/chezou/tabula-py/blob/v2.8.2/pyproject.toml#L2

Reply:
Thanks, done.